src/main/java/org/example/bookshop/controller/BookController.java

```diff
@@ -26,7 +26,7 @@
 import org.springframework.web.bind.annotation.RestController;
 
 @RestController
-@RequestMapping("/books")
+@RequestMapping("/api/books")
```

Comment:
you can add `server.servlet.context-path=/api` to application.properties
```suggestion
@RequestMapping("/books")
```